s3psetup/s3psetup/cortx_s3_py_setup.py

```diff
@@ -0,0 +1,154 @@
+#
```

Comment:
I think you should add the s3psetup directory to cortx-s3server/scripts/provisioning